extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -265,13 +282,16 @@ export class SchemaCompareDialog {
 		let currentButton = isTarget ? this.targetFileButton : this.sourceFileButton;
 
 		currentButton.onDidClick(async (click) => {
+			// file browser should open where the current dacpac is or the appropriate default folder
 			let rootPath = vscode.workspace.rootPath ? vscode.workspace.rootPath : os.homedir();
+			let defaultUri = endpoint && endpoint.packageFilePath ? endpoint.packageFilePath : rootPath;
```

Comment:
Should we check folder path for packageFilePath exists or file browser has a fallback behavior for that?

Reply:
added check so it will use default folder if the file doesn't exist